cl/api/templates/rest-docs-vlatest.html

```diff
@@ -13,8 +13,29 @@
 {% endblock %}
 
 {% block content %}
-  <div class="col-sm-7">
+  <div class="col-sm-3">
+
+    {% include "includes/toc_sidebar.html" %}
+
+  </div>
```

Comment:
This should copy some of the other templates, and has extra whitespace, so:

```suggestion
  <div class="hidden-xs hidden-sm col-md-3">
    {% include "includes/toc_sidebar.html" %}
  </div>
```